jsk_fetch_robot/jsk_fetch_user.rosinstall.melodic

```diff
@@ -0,0 +1,7 @@
+# This file is for work space in indigo user PC.
```

Comment:
indigo -> melodic?

Reply:
Thank you for your review.
I fixed the typo.